did-endpoint/README.md

```diff
@@ -0,0 +1,164 @@
+# did-endpoint 
+## Overview
+The `did-endpoint` is a robust, flexible, and extendable framework for managing Decentralized Identifiers (DIDs)
+
+## Purpose
+The did-endpoint aims to simplify the management of DIDs by providing a centralized service that supports multiple DID methods
+
+## Features
+- **Turns an HTTP(S) URL into a did:web id:** 
+- **Generates keys and forward them for DID generation:**
+- **Builds and persists DID document:**
+- **Validates the integrity of the persisted diddoc:**
+- **generates a verifiable presentation (VP):**
+
+## Usage
+- **Turns an HTTP(S) URL into a did:web id:** 
+```rust
+can_url_to_did_web_id() -> Result<(), Box<dyn Error>> {
+        assert_eq!(
+            url_to_did_web_id("localhost:8080")?,
+            "did:web:localhost%3A8080",
+        );
+}
```

Comment:
This is a test actually the real function to change HTTP URL to did:web id is ```url_to_did_web_id``` use it instead